docs/source/frida_bindings.rst

```diff
@@ -1,7 +1,7 @@
 Frida bindings
 ==============
 
-We propose bindings for Frida, the whole C/C++ API is available through them, but they are also backed up by helpers that fluidify script writing.
+We propose bindings for Frida, the whole C/C++ API is available through them, but they are also backed up by helpers that fluidifies script writing.
```

Comment:
I think 'helpers that fluidify' was fine.

Reply:
Indeed, my bad.